Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM opensciencegrid/software-base:3.6-el8-release
+RUN yum install -y openssl-devel bzip2-devel libffi-devel wget
+RUN yum groupinstall -y "Development Tools"
+RUN wget https://www.python.org/ftp/python/3.11.4/Python-3.11.4.tgz
```

Comment:
There's gotta be a better way to install python.  Look through `dnf search python` to see if there is a newer version of python.